build_image.sh

```diff
@@ -6,6 +6,7 @@ KEYRING="cuda-keyring_1.0-1_all.deb"
 CUDA_REPO_INSTALLER="cuda-repo-ubuntu2004-11-8-local_11.8.0-520.61.05-1_amd64.deb"
```

Comment:
I'm so confused. Maybe I'm super tired. Okay so we ARE using CUDA 11.8 but gcc-9??? Ugh too tired to fix my previous comments. Sowwy